processor/k8sprocessor/kube/client.go

```diff
@@ -51,19 +51,21 @@ type WatchClient struct {
 	Rules        ExtractionRules
 	Filters      Filters
 	Associations []Association
+	Exclude      ExcludePods
```

Comment:
This should refer to a type that represents the top level exclude config and has a field called `Pods` of type `ExcludePods`.